api/src/main/java/pl/kolendateam/dadcard/characterCard/entity/Character.java

```diff
@@ -182,13 +183,16 @@ public void createSkillsArray(List<Skills> skillsList) {
         for(int x = 0; x < skillsList.size(); x++){
```

Comment:
tu sie lepies sprawdzi ten for: for (Typ pojedynczawartosv : twojatablica) .  Unikaj jako iteratora x,y,z bo sugeruja ze robisz cos w kartezianskim. stosuje sie i oraz j.